### PR TITLE
Show indication when no labels are found

### LINKED ISSUE #164: No indication that there are no labels found
**Is your feature request related to a problem? Please describe.**

When using the filter input to search for specific labels, if none of the labels match the search text, there is no indication that there are no labels found.

**Describe the solution you'd like**

An indicator like `No Labels Found` to be displayed when no labels match the search text.

**Describe alternatives you've considered**

This feature is good to have, but not required.

**Additional context**

image

### PR DESCRIPTION
**Pull request by seetohjinwei**
### Summary:

Fixes #164 

#### Type of change:

- ✨ New Feature/ Enhancement

### Changes Made:

- Show indication of "No Labels Found!" when no labels are found.
- The styling is chosen to look similar to the button above.

### Screenshots:

https://github.com/CATcher-org/WATcher/assets/47915290/e4b817c8-d907-4663-bd30-3563d57de259

### Proposed Commit Message:

```
Show indication when no labels are found

Previously, when no labels are found, there was no indication.

Let's
* Show an indication when no labels match the input
* Show an indication when the repository has no labels
```

<details><summary>
<h3>Checklist:</h3>
</summary>

- [X] I have tested my changes thoroughly.
- [ ] I have created tests for any new code files created in this PR or provided a link to a issue/PR that addresses this.
- [X] I have added or modified code comments to improve code readability where necessary.
- [ ] I have updated the project's documentation as necessary.

</details>

### CODE REVIEW

File: src/app/shared/filter-bar/label-filter-bar/label-filter-bar.component.ts [hunk 1]
```diff
@@ -95,6 +95,13 @@ export class LabelFilterBarComponent implements OnInit, OnDestroy {
     return !target.toLowerCase().includes(filter.toLowerCase());
   }
 
+  hasNoLabels(filter: string): boolean {
+    if (this.allLabels === undefined || this.allLabels.length === 0) {
+      return true;
+    }
+    return this.allLabels.every((label) => this.filter(filter, label.name));
```
Comment (gycgabriel) [inline src/app/shared/filter-bar/label-filter-bar/label-filter-bar.component.ts:102]: Prefer to use .some and invert the boolean

As long as one label is shown, do not show the No Labels text

**Review by gycgabriel — CHANGES_REQUESTED** [review]
Just a small nitpick

File: src/app/shared/filter-bar/label-filter-bar/label-filter-bar.component.ts (hunk 1 above)
Comment (seetohjinwei) [reply src/app/shared/filter-bar/label-filter-bar/label-filter-bar.component.ts]: I've made the change!

**Review by gycgabriel — APPROVED** [review]
LGTM

**Pull request merged**